src/pages/Timetable/components/MyLectureTimetable/index.tsx

```diff
@@ -42,6 +41,14 @@ export default function MyLectureTimetable() {
           <img src="https://static.koreatech.in/assets/img/ic-image.png" alt="이미지" />
           이미지로 저장하기
         </button>
+        <button
+          type="button"
+          className={styles.page__button}
+          onClick={() => navigate('/timetable/modify')}
+        >
+          <img src="https://static.koreatech.in/assets/img/ic-image.png" alt="이미지" />
```

Comment:
이런건 버튼이 접근성을 담당해주니 대체텍스트가 불필요할 것 같네요~
```suggestion
          <img src="https://static.koreatech.in/assets/img/ic-image.png" alt="" />
```

Reply:
수정했습니다 089859169042a88faffff596642d82949bf17a58